packages/dso-toolkit/src/components/alert/alert.mixins.scss

```diff
@@ -78,14 +78,41 @@
   @media screen and (min-width: #{media-query-breakpoints.$screen-xs-min + 1}) {
     padding-block: alert-variables.$block-padding;
     padding-inline: alert-variables.$inline-start-padding alert-variables.$inline-padding;
+
+    &.dso-small {
+      padding-block: alert-variables.$block-padding * 0.5;
+      padding-inline: alert-variables.$inline-padding;
+    }
   }
 
   @media screen and (max-width: media-query-breakpoints.$screen-xs-min) {
     padding-block: alert-variables.$small-padding-block-start alert-variables.$small-padding;
     padding-inline: alert-variables.$small-padding;
+
+    &.dso-small {
+      padding-block: alert-variables.$block-padding * 0.5;
+      padding-inline: alert-variables.$inline-padding;
+    }
   }
 
   &:not(:first-child) {
     margin-block-start: units.$u3;
   }
 }
+
+@mixin small($bottom: 0, $top: 0) {
+  border-bottom-right-radius: units.$u1;
+  border-top-right-radius: units.$u1;
+  margin-inline-start: units.$u1;
+
+  &::before {
+    border-bottom-left-radius: units.$u1;
+    border-top-left-radius: units.$u1;
+    bottom: $bottom;
+    content: "";
+    left: units.$u1 * -1;
+    position: absolute;
+    top: $top;
+    width: units.$u1;
```

Comment:
Zou dit niet `inline-size` moeten zijn? Waarom faalt de linter hier niet op?

Reply:
Er staan nog meer physical properties in deze selector 🤔

Reply:
Nadat ik lokaal de master gereset heb naar de commit, voordat #2366 erin is gemerged, heb ik `stylelint-use-logical-spec` en configuratie ervoor toegevoegd en vervolgens `yarn lint:stylelint` gedraaid. Ik zie dan alleen scss-files uit de core package terug in de lijst van 455 errors. Package `dso-toolkit` wordt niet gescand? In `package.json`: `"lint:stylelint": "stylelint 'packages/**/*.scss'",`. Men zou verwachten dat dat glob-pattern ook `packages/dso-toolkit` zou omvatten.

Reply:
Hier ga ik een los issue voor maken, laat voor nu de physical properties maar staan zoals ze staan.

Reply:
@hansgrimm Zie #2714 voor jouw opmerkingen over logical vs. physical properties.

@iterox zoals besproken: laat het in dit PR maar staan. Hans gaat dat straks fixen